src/components/MobileHeader/MobileHeader.js

```diff
@@ -0,0 +1,46 @@
+import React, { Fragment } from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import { Link } from 'react-router-dom'
+import { Button, Icon } from '@santiment-network/ui'
+import styles from './MobileHeader.module.scss'
+
+const MobileHeader = ({
+  title,
+  backRoute,
+  rightActions,
+  classes = {
```

Comment:
@Partysun would be better if you define a const (may be `defaultClasses` or some better name) at the top and export it. Would be possible to extend from parent component without providing all the styles